src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpCwdsDAO.scala

```diff
@@ -69,5 +101,14 @@ class HttpCwdsDAO(enabled: Boolean) extends CwdsDAO {
     STATUS_TRANSLATION.getOrElse(cwdsStatus, "Unknown")
   }
 
+  private def getApiClient(accessToken: String): ApiClient = {
+    // prepare the cWDS client
+    val apiClient: ApiClient = new ApiClient()
+    apiClient.setHttpClient(commonHttpClient)
+    apiClient.setBasePath(FireCloudConfig.Cwds.baseUrl)
+    apiClient.setAccessToken(accessToken)
```

Comment:
Just wanna double check...is this the right token?

Reply:
how many times can I make the same mistake?!??!?! ARRGARGAF